### PR TITLE
renamed DSX to Watson Studio

### PR DESCRIPTION
**Pull request by sanjeevghimire**

### CODE REVIEW

File: README-ko.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
-# Watson Natural Language Understanding과 IBM Data Science Experience를 활용하여 텍스트 분류 보강하기
+# Watson Natural Language Understanding과 IBM Watson Studio를 활용하여 텍스트 분류 보강하기
 
 *Read this in other languages: [English](README.md).*
 
-이 개발자 과정에서는 IBM Data Science experience(DSX)에서 주피터 노트북을 사용하여 IBM Watson Natural Language Understanding API의 분석 결과를 텍스트 분류를 위한 구성 가능한 메커니즘을 통해 개선합니다.
+이 개발자 과정에서는 IBM Data Science experience(Watson Studio)에서 주피터 노트북을 사용하여 IBM Watson Natural Language Understanding API의 분석 결과를 텍스트 분류를 위한 구성 가능한 메커니즘을 통해 개선합니다.
```
Comment (scottdangelo) [inline README-ko.md]: remove 'Data Science experience'

File: README-ko.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
 이 개발자 과정을 완료하면 다음을 이해할 수 있습니다:
 
-* DSX에서 주피터 노트북을 생성 및 실행합니다.
-* DSX Object Storage를 사용하여 데이터 및 구성 파일에 접근합니다.
+* Watson Studio에서 주피터 노트북을 생성 및 실행합니다.
+* Watson Studio Object Storage를 사용하여 데이터 및 구성 파일에 접근합니다.
```
Comment (scottdangelo) [inline README-ko.md]: not sure about rename of Object Storage

File: README-ko.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 * 주피터 노트북에서 NLU API를 사용하여 문서의 메타데이터를 추출합니다.
 * 간단한 Python 함수로 비정형 데이터를 추출하고 형식을 지정 합니다.
 * 구성 파일을 사용하여 구성 가능하고 계층화된 분류 문법을 작성합니다.
 * 구성 파일의 문법적 분류와 정규식 패턴을 조합하여 단어의 토큰 클래스를 분류합니다.
-* 처리 결과를 JSON 형식으로 DSX Object Storage에 저장합니다.
+* 처리 결과를 JSON 형식으로 Watson Studio Object Storage에 저장합니다.
```
Comment (scottdangelo) [inline README-ko.md]: not sure

File: README-ko.md [hunk 4]
```diff
@@ -48,9 +48,9 @@
 1. [결과 다운로드](#7-결과-다운로드)
 1. [결과 분석](#8-결과-분석)
 
-## 1. Data Science Experience 가입
+## 1. Watson Studio 가입
 
-IBM [Data Science Experience](http://datascience.ibm.com/)에 가입하면 블루믹스 계정에 두개의 서비스(``DSX-Spark``와 ``DSX-ObjectStore``)가 생성됩니다.
+IBM [Watson Studio](http://datascience.ibm.com/)에 가입하면 블루믹스 계정에 두개의 서비스(``Watson Studio-Spark``와 ``Watson Studio-ObjectStore``)가 생성됩니다.
```
Comment (scottdangelo) [inline README-ko.md]: not sure these changed, and to what

File: README-ko.md [hunk 5]
```diff
@@ -149,7 +149,7 @@ UI에 열려있는 `Service Credentials`의 `KEY NAME`열에서 노트북에서
 
 ## 7. 결과 다운로드
 
-* 결과를 보려면 [DSX-ObjectStore](https://console.bluemix.net/dashboard/storage) 사이트를 방문합니다.
+* 결과를 보려면 [Watson Studio-ObjectStore](https://console.bluemix.net/dashboard/storage) 사이트를 방문합니다.
```
Comment (scottdangelo) [inline README-ko.md]: not sure

File: README.md [hunk 6]
```diff
@@ -2,26 +2,26 @@
 
 *Read this in other languages: [한국어](README-ko.md).*
 
-In this developer journey we will use Jupyter notebooks in IBM Data Science experience(DSX) to augment
+In this developer journey we will use Jupyter notebooks in IBM Data Science experience(Watson Studio) to augment
```
Comment (scottdangelo) [inline README.md]: remove Data Science experience

File: README.md [hunk 7]
```diff
@@ -2,26 +2,26 @@
...
 IBM Watson Natural Language Understanding API output through configurable mechanism for text classification.
 
 When the reader has completed this journey, they will understand how to:
 
-* Create and run a Jupyter notebook in DSX.
-* Use DSX Object Storage to access data and configuration files.
+* Create and run a Jupyter notebook in Watson Studio.
+* Use Watson Studio Object Storage to access data and configuration files.
```
Comment (scottdangelo) [inline README.md]: not sure

File: README.md [hunk 8]
```diff
@@ -2,26 +2,26 @@
...
 * Use IBM Watson Natural Language Understanding API to extract metadata from documents in Jupyter notebooks.
 * Extract and format unstructured data using simplified Python functions.
 * Use a configuration file to build configurable and layered classification grammar.
 * Use the combination of grammatical classification and regex patterns from a configuration file to classify word token classes.
-* Store the processed output JSON in DSX Object Storage.
+* Store the processed output JSON in Watson Studio Object Storage.
```
Comment (scottdangelo) [inline README.md]: not sure

File: README.md [hunk 9]
```diff
@@ -162,7 +162,7 @@ There are several ways to execute the code cells in your notebook:
 
 ## 7. Download the results
 
-* To see the results, go to [DSX-ObjectStore](https://console.bluemix.net/dashboard/storage)
+* To see the results, go to [Watson Studio-ObjectStore](https://console.bluemix.net/dashboard/storage)
```
Comment (scottdangelo) [inline README.md]: not sure

**Review by scottdangelo — DISMISSED** [review]

**Review by stevemar — APPROVED** [review]

**Pull request merged**